README.md

```diff
@@ -6,7 +6,7 @@ Finesse is a project designed to provide users with the ability to search for ag
 
 ## Video Demonstration
 
-[Finesse Demo Video](https://youtu.be/Bw6_azllnDw?si=-eO0BZQmi_iHXNr1)
+[Finesse Demo Video](https://youtu.be/7sDQ9smVMww?si=uSTCZKUUrhT5vbcc)
```

Comment:
?si= is a tracking token. don't check it in.

Reply:
@CFIALeronB why is this still pending?

Reply:
> @CFIALeronB why is this still pending?

Tracking token removed.